lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -266,13 +294,14 @@ $notes-annotator-background-dark: rgba(122,122,122,0.6); // taken from annotator
 
     // controls
     .annotator-controls {
+      opacity: 1;
       // RTL support
       @include right(0);
```

Comment:
@clrux First, `@include`, `@expend` should be, and then followed by other CSS rules.